apps/daimo-mobile/app.json

```diff
@@ -76,10 +76,16 @@
       [
         "expo-local-authentication",
         {
-          "faceIDPermission": "Protect Daimo using Face ID."
```

Comment:
App review didn't mention Face ID, right?

Reply:
nope, but their account onchain they hasn't gotten to that prompt yet (no sends): https://goerli.basescan.org/address/0x8f5c3b22f7d54bea27fba16de042094ac4201140

so i'm just updating pre-emptively since the old string is the default expo auto-generated one.